frontend/src/app/testResults/TestResultsList.tsx

```diff
@@ -119,6 +123,9 @@ export const DetachedTestResultsList: any = ({
             "sr-test-result-row",
             removed && "sr-test-result-row--removed"
           )}
+          data-patient-link={`${getUrl()}pxp?plid=${
+            r?.patientLink?.internalId
```

Comment:
`r` already exists so you probably don't need the first optional chaining. The second one for `patientLink?.internalId` I'm less sure about but I expect we could assume it exists. But if there's a chance `r.patientLink` _doesn't_ exist, I have to imagine this isn't the most graceful degradation: you'd have a broken pxp link.

Reply:
You're right - this was just laziness on my part. I think it is possible that `r.patientLink` won't exist for older results submitted before this PR is merged, so instead I should check for that, and if it isn't present, don't include a `data-patient-link` prop at all. Thanks for calling this out